### PR TITLE
[SPA-173] Add info label - Improve component

### PR DESCRIPTION
**Pull request by robergro**
- Remove unnecessary dependencies.
- Rename **description** to **helper** and add deprecated init and properties.
- Add **info label** on UIKit.
- Remove UIControl inheritance on UIKit and properties (isEnabled, isHighlighted, isSelected).
- Update snapshots.

### CODE REVIEW

File: Sources/Core/View/SwiftUI/FormFieldView.swift [hunk 1]
```diff
@@ -11,39 +11,70 @@ import SparkTheming
...
+
+    /// Initialize a formField.
+    /// - Parameters:
+    ///   - theme: The current Spark-Theme.
+    ///   - component: The component is covered by formfield.
+    ///   - feedbackState: The formfield feedback state. 'Default' or 'Error'.
+    ///   - title: The formfield title.
+    ///   - description: The formfield helper message.
+    ///   - isTitleRequired: The asterisk symbol at the end of title.
+    @available(*, deprecated, message: "Replaced by the init with the helper String since the 0.1.1. (15/10/2024)")
```
Comment (LouisBorleeAdevinta) [inline Sources/Core/View/SwiftUI/FormFieldView.swift:58]: not a fan of the date

File: Sources/Core/ViewModel/FormFieldViewModel.swift [hunk 2]
```diff
@@ -56,47 +62,82 @@ final class FormFieldViewModel<AS: SparkAttributedString>: ObservableObject {
...
 
     private func updateSpacing() {
-        self.spacing = self.theme.layout.spacing.small
+        self.spacing = Self.spacing(from: self.theme)
+    }
+
+    // MARK: - Static func
+
+    private static func titleFont(from theme: Theme) -> any TypographyFontToken {
+        theme.typography.body2
```
Comment (LouisBorleeAdevinta) [inline Sources/Core/ViewModel/FormFieldViewModel.swift:129]: don't forget returns

**Review by LouisBorleeAdevinta — COMMENTED** [review]
I think the labels (helper and info) should be made public so we wouldn't have to handle things in initializers

Also now that I think about it, helper and info are kind of synonyms, maybe we could find another wording like "supplementaryHelper" or I don't know

File: Sources/Core/ViewModel/FormFieldViewModel.swift (hunk 2 above)
Comment (robergro) [reply Sources/Core/ViewModel/FormFieldViewModel.swift:129]: I don't like this rule but Ok, I made the change :)

File: Sources/Core/View/SwiftUI/FormFieldView.swift (hunk 1 above)
Comment (robergro) [reply Sources/Core/View/SwiftUI/FormFieldView.swift:58]: Ok ! There is no date anymore

**Review by LouisBorleeAdevinta — APPROVED** [review]

**Pull request merged**